src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -472,13 +497,13 @@ protected void waitUntilContainerStarted() {
         }
     }
 
-    protected void waitForBucketFs(final Instant afterUtc) {
+    protected void waitForBucketFs() {
         if (isServiceReady(BUCKETFS)) {
             LOGGER.debug("BucketFS marked running in container status cache. Skipping startup monitoring.");
         } else {
             if (this.requiredServices.contains(BUCKETFS)) {
                 this.status.setServiceStatus(BUCKETFS, NOT_READY);
-                new BucketFsWaitStrategy(this.detectorFactory, afterUtc).waitUntilReady(this);
+                new BucketFsWaitStrategy(this.detectorFactory).waitUntilReady(this);
```

Comment:
are you sure you can remove it?

Reply:
Yes. This is the same issue as with `UdfContainerWaitStrategy`. We only want to check if the expected log entry is there and ignore the timestamp. Else we can get timing issues.